.github/workflows/package.yml

```diff
@@ -69,17 +69,23 @@ jobs:
         label:
           - CentOS 7
           - Debian GNU/Linux bullseye
+          - AlmaLinux 9
         include:
           - label: CentOS 7
             id: centos-7
             task-namespace: yum
             target: centos-7
             test-docker-image: centos:7
-          - label: Debian GNU/Linux bullseye
-            id: debian-bullseye
-            task-namespace: apt
-            target: debian-bullseye
-            test-docker-image: debian:bullseye
```

Comment:
Why is this block removed? (To reduce CI time?)

Reply:
Ah, It's just a mistake. Will revert it.